commercetools-models/src/main/java/io/sphere/sdk/categories/commands/updateactions/ChangeAssetOrder.java

```diff
@@ -0,0 +1,22 @@
+package io.sphere.sdk.categories.commands.updateactions;
+
+import io.sphere.sdk.categories.Category;
+import io.sphere.sdk.commands.UpdateActionImpl;
+
+import java.util.List;
+
+/**
+ * Changes the order of the assets list. The new order is defined by listing the ids of the assets.
+ */
+public final class ChangeAssetOrder extends UpdateActionImpl<Category> {
+    private final List<String> assetOrder;
+
+    public ChangeAssetOrder(final List<String> assetOrder) {
```

Comment:
Constructor should be private and there should be a static constructor available (`of` method).